### ISSUE TITLE
Add multi-accounts support

### ISSUE THREAD

**Issue by ShGKme** [nextcloud/talk-desktop#7]

**Comment by AedDev** [comment]
I'd also love to see multi-account being added to this great piece of software. NextCloud Enterprise Hosting is part of my daily work and my customers, who use Talk inside NextCloud, can also reach out to the customer support via Talk. Having all instances combined in a single messenger would be a huge increase in efficiency.

**Comment by ShGKme** [comment]
@AedDev As a workaround with a portable (`.zip`) distribution, you can rename the executable to run multiple instances simultaneously. See: https://github.com/nextcloud/talk-desktop?tab=readme-ov-file#-multi-account

While Multi-account support is an important feature, it is not currently planned for 1.0 release in flavor of more critical features/fixes without workarounds.

**Comment by AedDev** [comment]
@ShGKme thank you for your response! It works this way, but with a lot of instances, it's getting tricky. Maybe as temporary workaround a collection of cli arguments could bring some more control. Like:

```
--instance-name "cloud.customer1.com"
--instance-icon "/path/to/icon.ico"
--hide-tray # since too many tray icons are pain
```

The name could also appear in the window title, the icon could be used for the tray menu as well as for the window. This way the different clients could be identified more easily.

Maybe I could also open up a PR for that feature, in case my dusty knowledge in the used languages and frameworks is still sufficent to implement changes like this.

**Comment by kernstock** [comment]
With the flatpak distribution, this would be a great feature!

**Comment by filozofer** [comment]
Where can I find the “windows” zip version of the application? On the README I only see: exe / msi / flat.

**Comment by ShGKme** [comment]
Good point. We'll return back `.zip` distribution until multi-account support is added.

**Comment by filozofer** [comment]
@ShGKme Still waiting for this Windows `.zip` distribution. Not included inside the v1.1.6 release ?

**Comment by sylvaticus** [comment]
Same wish here.. started using NC Talk at work, and loved it so much that I started using it for personal usage with my family (on a different instance), so I need multiple account support now  :-) :-)

**Comment by drlight17** [comment]
> Same wish here.. started using NC Talk at work, and loved it so much that I started using it for personal usage with my family (on a different instance), so I need multiple account support now :-) :-)

Try[ this pre-release](https://github.com/drlight17/talk-electron/releases/tag/1.0.0-RC)

**Comment by realaaa** [comment]
+1 from down under here 

we are using Nextcloud more and more - and having multi-accounts natively is quickly becoming a necessity now

in my case I am on Mac OS, but of course this is cross platform requirement for Windows and Linux as well

Happy to help with testing etc

**Comment by vasyugan** [comment]
Maybe, I'm overlooking something, but without multi-account support (which the  Android client has been having for years already), I can't see a significant added value of the desktop client over talk in a browser tab.

**Comment by drlight17** [comment]
> Maybe, I'm overlooking something, but without multi-account support (which the Android client has been having for years already), I can't see a significant added value of the desktop client over talk in a browser tab.

Try[ this ](https://github.com/drlight17/talk-electron/releases/latest) while waiting official support ;) I've developed it for my multiple jobs with NC and Talk

**Comment by DavidWikstrom** [comment]
This would be a huge game changer for me. My companies has a couple of different NextCloud instances, and several of our clients use it too, so it would be so great to be able to have all conversations in one place.
